relay.ts

```diff
@@ -12,7 +12,7 @@ export type Relay = {
   close: () => Promise<void>
   sub: (filters: Filter[], opts?: SubscriptionOptions) => Sub
   publish: (event: Event) => Pub
-  on: (type: RelayEvent, cb: any) => void
+  on: (type: RelayEvent, cb: (event: Event)=>void) => void
```

Comment:
This is not correct because it doesn't apply when the event received is a `connect`, `disconnect` or `notice`.